### ISSUE TITLE
Ranged Attack problem

### ISSUE THREAD

**Issue by Xenith-the-Red** [EdenServer/community#1539]
### Checklist

- [/] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [/] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [/] I have provided my client version in the details. (type /ver into your game window)

### Details

**please note the following issue applies to my experience with Bow+Arrow, but I imagine it applies to xBow+bolts too**

version 30181205_0

Problem:
The message "You must wait longer to perform that action" inappropriately appears when reattempting a ranged attack, after a failed one (for a time comparable to the delay of the respective ranged weapon).

Examples of normal failures:
"You cannot see the monster"
"The monster is too far away"
"You do not have an appropriate ranged weapon equipped"

The first example can be tested by attempting to /ra a monster while facing away from it, and immediately correcting your line of sight to attempt another /ra.

The second example can be tested by attempting to /ra a monster from 25 yalms away, and immediately correcting your position to 24 yalms away to attempt another /ra.

The third example can be tested by attempting to Bow-attack a monster without arrows equipped (but held in inventory), and immediately (and successively) using a macro which combines an /ra attack line, and an arrow-equip line.

In summary:
There shouldn't be a delay-based cooldown after a "small" ranged failure.  Players should be able to correct their line-of-sight, distance from monster, or ammo, and be immediately able to use /ra thereafter.

**Comment by GalkaZajin** [comment]
Duplicate to #859 and many others

**Comment by 59blargedy** [comment]
this was resolved in early 2021
